lib/qiita/markdown/filters/greenmat.rb

```diff
@@ -16,13 +20,18 @@ def greenmat
             Qiita::Markdown::Greenmat::HTMLRenderer.new(hard_wrap: true, with_toc_data: true),
             autolink: true,
             fenced_code_blocks: true,
-            footnotes: true,
+            footnotes: options[:footnotes],
             no_intra_emphasis: true,
             no_mention_emphasis: true,
             strikethrough: true,
             tables: true,
           )
         end
+
+
+        def options
+          @options ||= DEFAULT_OPTIONS.merge(context[:greenmat] || {})
+        end
```

Comment:
Since `Processor` uses `Filters::Greenmat` too, doesn't this change affect to the default behavior of `Processor`? We want to keep the default behavior (i.e. using `Qiita::Markdown::Processor` without customization) to match Qiita's Markdown spec.